.pylintrc

```diff
@@ -46,7 +46,7 @@ disable=
 
 good-names=
     # PyLint's default good names.
-    i,j,k,ex,Run,_,
+    i,j,k,s,ex,Run,_,
```

Comment:
Instead of doing this I think we'd be better off just picking a sensible variable name like `string`

Reply:
You don't want to use `string` as a variable name because there's a standard library module called `string`. I think `s` is reasonable as a common variable name for functions that just work on a single string. Otherwise maybe `string_`

Reply:
This is basically the same code as Pyramid's `asbool`, yes, including the `s`. Pyramid's `asbool` has some special handling of booleans that there's probably a reason for but we don't need here:

```python
def asbool(s):
    """Return the boolean value ``True`` if the case-lowered value of string
    input ``s`` is a :term:`truthy string`. If ``s`` is already one of the
    boolean values ``True`` or ``False``, return it."""
    if s is None:
        return False
    if isinstance(s, bool):
        return s
    s = str(s).strip()
    return s.lower() in truthy
```

Reply:
Any sensible value would do here. `value` or something vague.

Reply:
I've changed this to `value` since actually it can be either a string or `None`. `s` really is a commonly-used variable name in Python functions that take a string as their only or primary argument, though